.github/workflows/check.yml

```diff
@@ -61,7 +61,7 @@ jobs:
             type: release
           # Also do some debug builds on the oldest OS versions.
           - os: ubuntu-22.04
-            rust-toolchain: stable
+            rust-toolchain: 1.81.0
```

Comment:
Without this change the Ubuntu 22.04 step fails with the below as noted by Lars above:

```
  /home/runner/work/mtu/mtu/target/release/build/clang-sys-da9bc230b96985b2/build-script-build: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.39' not found (required by /home/runner/work/mtu/mtu/target/release/build/clang-sys-da9bc230b96985b2/build-script-build)
```

https://github.com/mozilla/mtu/actions/runs/15249871587/job/42903683731

I don't understand why. The MSRV should not impact how `mtu` is build with a `rustc` that is larger than the MSRV (i.e. here `stable` which is `1.87.0`).

@larseggert any ideas? I will debug this some more tomorrow.